### PR TITLE
Hotfix/docker openwrt builder

### PR DESCRIPTION
**Pull request by tomereli**

### CODE REVIEW

**Review by ghost — APPROVED** [review]

File: tools/docker/builder/openwrt/Dockerfile
Comment (arnout) [inline tools/docker/builder/openwrt/Dockerfile]: Why do you remove the indentation here? It *was* properly indented inside the condition...

File: tools/docker/builder/openwrt/Dockerfile
Comment (arnout) [inline tools/docker/builder/openwrt/Dockerfile:71]: I think this was done to allow the continuation backslash to work.However, apparently it's not needed... At least not in CI and not on my own machine. @rmelotte please confirm.

**Review by arnout — APPROVED** [review]
Please fix the indentation, and wait for @rmelotte approval.

File: tools/docker/builder/openwrt/Dockerfile
Comment (rmelotte) [inline tools/docker/builder/openwrt/Dockerfile]: As Arnout said I think it's clearer to keep the indentation for the if-else blocks

File: tools/docker/builder/openwrt/Dockerfile
Comment (rmelotte) [reply tools/docker/builder/openwrt/Dockerfile:71]: I had the impression that the '#' character had to be at column 1. 
Apparently it's not the case anymore since 2015 :neutral_face:
https://github.com/docker/cli/commit/869e08a12bf3bf614e74af34f46a6c1d3d061afb

**Review by rmelotte — APPROVED** [review]

**Comment by tomereli** [comment]
The first commit is auto indenting the code when I save it in vscode, it was done by @vitalybu.

File: tools/docker/builder/openwrt/Dockerfile
Comment (tomereli) [reply tools/docker/builder/openwrt/Dockerfile]: This is the default docker formatter in vscode, so apparently it wasn't properly indented according to its default rules.

**Comment by vitalybu** [comment]
> The first commit is auto indenting the code when I save it in vscode, it was done by @vitalybu.

image

File: tools/docker/builder/openwrt/Dockerfile
Comment (tomereli) [reply tools/docker/builder/openwrt/Dockerfile]: I reverted this change, but note that it is *against* the formatting rules so if someone will re-format this file it will be done again.

File: tools/docker/builder/openwrt/Dockerfile
Comment (tomereli) [reply tools/docker/builder/openwrt/Dockerfile]: Done, but as I said - its against the formatting rules.

File: .vscode/settings.json
Comment (vitalybu) [inline .vscode/settings.json]: Please don't, it's an excellent feature.

**Review by vitalybu — CHANGES_REQUESTED** [review]

**Review by vitalybu — APPROVED** [review]

**Pull request merged**
